lib/aruba/cucumber/command.rb

```diff
@@ -28,6 +28,21 @@
   run_simple(cmd, :fail_on_error => true, :exit_timeout => secs && secs.to_i)
 end
 
+When(/I run the following script:$/) do |file_content|
+  prepend_environment_variable('PATH', expand_path('bin') + ':')
```

Comment:
Mmmh... Can you try to invoke your script with `./bin/myscript` in the run step and leave out the prepend api call? If I remember correctly aruba accepts relative paths.